packages/salesforcedx-vscode-core/src/context/workspaceOrgType.ts

```diff
@@ -16,30 +18,30 @@ export enum OrgType {
 export async function getWorkspaceOrgType(
   defaultUsernameOrAlias?: string
 ): Promise<OrgType> {
-  if (!defaultUsernameOrAlias) {
-    const e = new Error();
-    e.name = 'NoDefaultusernameSet';
-    throw e;
-  }
-  const username = await OrgAuthInfo.getUsername(defaultUsernameOrAlias);
-  const isScratchOrg = await OrgAuthInfo.isAScratchOrg(username).catch(err =>
-    telemetryService.sendException(
-      'get_workspace_org_type_scratch_org',
-      err.message
-    )
-  );
-  return isScratchOrg ? OrgType.SourceTracked : OrgType.NonSourceTracked;
+  const connection = await WorkspaceContext.getInstance().getConnection();
+  const org: Org = await Org.create({ connection });
+  // Org.supportsSourceTracking() checks the org at this point in time.  This
```

Comment:
Or if it is worth keeping in code, could we move to a more traditional function-level /**/ comment?

Reply:
Since we know that refreshing a Sb requires re-auth, I'm going to switch this back to use tracksSource().  If it isn't behaving as expected we will see that in QA.